test/integration/targets/filesystem/tasks/setup.yml

```diff
@@ -1,10 +1,12 @@
-- name: install e2fs and xfs
+- name: install filesystem tools
   package:
     name: '{{ item }}'
     state: present
+  when: ansible_system != 'FreeBSD' or item != 'dosfstools'
```

Comment:
I am not sure to understand that conditional. Why do you add dosfstools if that's excluded each time ?

Reply:
It's not excluded when `ansible_system != 'FreeBSD'` is `True`.